src/components/card/Card.jsx

```diff
@@ -0,0 +1,36 @@
+export const Card = () => {
+  return (
+    <div class="card">
```

Comment:
In react you have to use className instead of class as class is reserved keyword in JavaScript as we are using JSX ,we use className attribute. it will give you warning in your console aswell.